services/server/src/wss/handlers/connection.ts

```diff
@@ -84,15 +84,21 @@ export class Connection extends WssHandler {
       if (isStudent(this.user)) this.socket.join(Wss.Room.TutorsCache);
       if (isAdmin(this.user)) this.socket.join(Wss.Room.ServerStats);
 
-      // todo: get user calls from cache
-      const callsList = await calls.find({
+      // todo: get user sessions from cache
+      const lessonsSessions = (await lessons.find({
         users: [user.id],
         full: true,
         after: dayjs.utc().startOf("day").toISOString(),
         before: dayjs.utc().add(1, "day").toISOString(),
-      });
+      })).list.map(lesson => lesson.sessionId);
+
+      const interviewsSessions = (await interviews.find({ 
+        users: [user.id] 
+      })).list.map(interview => interview.ids.session);
 
-      this.socket.join(callsList.list.map((call) => asCallRoomId(call.id)));
+      this.socket.join([...lessonsSessions, ...interviewsSessions].map(
+        (session) => asSessionRoomId(session))
+      );
     });
```

Comment:
We can remove this code here and we can thinkg abvout it more deeply later.

Reply:
We may put a comment about that and leave it as it is for now; there are dependencies on it (three test units fail without it).

Reply:
Aha, got it. ok, sounds good.